doc/conseil.sql

```diff
@@ -89,7 +89,17 @@ CREATE TABLE public.blocks (
     operations_hash character varying,
     period_kind character varying,
     current_expected_quorum integer,
-    active_proposal character varying
+    active_proposal character varying,
+    baker character varying,
+    nonce_hash character varying,
+    consumed_gas numeric,
+    meta_level integer,
+    meta_level_position integer,
+    meta_cycle integer,
+    meta_cycle_position integer,
+    meta_voting_period integer,
+    meta_voting_period_position integer,
+    meta_commitment boolean
```

Comment:
Could we rename this to `expected_commitment` please in order to keep the columns consistent with the Tezos node?

Reply:
absolutely, will do